src/ModelContextProtocol/Shared/McpSession.cs

```diff
@@ -590,8 +653,9 @@ public void Dispose()
         if (durationMetric.Enabled)
         {
             TagList tags = default;
-            tags.Add("session.id", _id);
             tags.Add("network.transport", _transportKind);
+
+            // MCP convention also includes: server.address, server.port, client.address, client.port
```

Comment:
Is the idea that we'll find a way to include that but only when using HTTP/SSE?

Reply:
yep, updated the comment with some corrections